horizons/gui/modules/settings.py

```diff
@@ -222,13 +222,13 @@ def slider_change(self, widget):
 		a '%' suffix."""
 		value_label = self.widget.findChild(name=widget.name + '_value')
 		value = {
-			'volume_music':      lambda x: '%s%%' % int(500 * x),
-			'volume_effects':    lambda x: '%s%%' % int(200 * x),
-			'mousesensitivity':  lambda x: '%+.1f%%' % (200 * x),
-			'autosaveinterval':  lambda x: '%d' % x,
-			'autosavemaxcount':  lambda x: '%d' % x,
-			'quicksavemaxcount': lambda x: '%d' % x,
-			'scrollspeed':       lambda x: '%.1f' % x,
+			'volume_music':      lambda x: '{:d}%%'.format(int(500 * x)),
+			'volume_effects':    lambda x: '{:d}%%'.format(int(200 * x)),
+			'mousesensitivity':  lambda x: '{:+.1f}%%'.format(200 * x),
```

Comment:
For the lines above - you don't have to double the percentage '%' sign in `.format() `string templates:
```py
>>> "{:d}%".format(200)
'200%'
>>> "{:d}%%".format(200)
'200%%'
>>> 
```